libcaf_net/caf/net/multiplexer.cpp

```diff
@@ -538,7 +540,7 @@ class default_multiplexer : public detail::atomic_ref_counted,
   /// later via the pollset updater.
   /// @warning assumes ownership of @p ptr.
   template <class T>
-  void write_to_pipe(uint8_t opcode, T* ptr) {
+  ptrdiff_t write_to_pipe(uint8_t opcode, T* ptr) {
```

Comment:
We don't really care about the value here. Only whether the write succeeded or not, so I'd return `bool` here as well.